src/arachne/pedestal/routes.clj

```diff
@@ -53,32 +53,44 @@
 
 (defn- route
   "Return a seq of Pedestal route maps for the given endpoint EID"
-  [router cfg eid]
+  [cfg eid]
   (for [method (cfg/attr cfg eid :arachne.http.endpoint/methods)]
     (let [path (http-cfg/route-path cfg (cfg/attr cfg eid :arachne.http.endpoint/route :db/id))]
       (merge (path/parse-path path)
-             {:route-name (cfg/attr cfg eid :arachne.http.endpoint/name)
-              :method method
-              :path path
-              :interceptors (map #(interceptor cfg % (get router %))
-                                 (interceptor-eids cfg eid))}))))
+             {:endpoint-eid eid
```

Comment:
I had some reservations about attaching arachne info to the route map but I found it to be the cleanest way to separate route generation from interceptor attachment. I'm definitely open to alternatives.